docker-compose.dev.yml

```diff
@@ -1,27 +1,82 @@
 services:
+  nginx:
```

Comment:
Q: what is the purpose of nginx in our dev environment

Reply:
isn't it better to simulate production env. also when we start including frontend i feel like it will be useful

Reply:
The purpose of this docker-compose is only for development, we wanted to make it as light-weighted as possible
the docker containers we are using in this docker-compose are to make updates reflected as soon as possible without any overhead on the developer's side

adding nginx may introduce some overhead in debugging

Reply:
We may set up another docker-compose in staging environment that uses nginx reverse proxy and set up CD hotfixes that can integrate the latest updates -> build/optimize -> then serve it using nginx and introduce some monitoring

For the purpose of development on developers' machine, I think nginx may be an overkill that makes development more complex